<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visitingu</class>
 <widget class="QWidget" name="visitingu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>448</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="teVisitU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twVisitU">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>12</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackVisitU">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Посещения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
